environments/centos7-python/requirements.txt

```diff
@@ -5,3 +5,4 @@ pandas==0.22.0
 scipy==1.0.0
 sympy==1.1.1
 Pillow==5.2.0
+pygraphviz==1.5
```

Comment:
To make this work in `centos7-python` I think you'll also need to add the yum packages to `centos7-python/Dockerfile`.